components/tabstrip/overview.md

```diff
@@ -10,35 +10,21 @@ position: 0
 
 # Blazor TabStrip Overview
 
-This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.
-
->caption In this article:
-<!-- Start Document Outline -->
-* [Creating Blazor TabStrip](#creating-blazor-tabstrip)
-	* [Position of the Tabs](#position-of-the-tabs)
-	* [Persist Content](#persist-content)
-	* [Scrollable Tabs](#scrollable-tabs)
-	* [Styling and Appearance](#styling-and-appearance)
-* [Example](#example)
-* [See Also](#see-also)
-
-<!-- End Document Outline -->---
+The <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> displays a collection of tabs, containing associated content, which enable the user to switch between different views inside a single component.
 
 ## Creating Blazor TabStrip
 
-To use a Telerik TabStrip for Blazor:
-
 1. Use the `<TelerikTabStrip>` tag.
 1. Add a nested `<TabStripTab>` tag for each tab you want to include in the component.
 1. Set tab titles through the `Title` parameter of the `<TabStripTab>` tags. You can additionally configure the tabs as desired - read more in the [Tabs Configuration]({%slug tabstrip-tabs-configuration%}) article.
```

Comment:
"You can additionally configure the tabs as desired" is not actionable and unnecessary. The link may be omitted too, as we link to that page below. `Visible` and `Enabled` are not vital parameters that justify the link here